Mathlib/Analysis/LocallyConvex/Bounded.lean

```diff
@@ -185,16 +191,8 @@ variable {𝕝 : Type*} [NormedField 𝕜] [NontriviallyNormedField 𝕝] [AddCo
 
 theorem IsVonNBounded.smul_tendsto_zero {S : Set E} {ε : ι → 𝕜} {x : ι → E} {l : Filter ι}
     (hS : IsVonNBounded 𝕜 S) (hxS : ∀ᶠ n in l, x n ∈ S) (hε : Tendsto ε l (𝓝 0)) :
```

Comment:
Looks like you can change `𝕜` to a `NormedDivisionRing` here as well.

Reply:
Indeed, more lemmas can be generalized. I have more in a local branch. I'm going to open a separate PR about that.